### ISSUE TITLE
Show alt text in media overlay in the Gallery screen

### ISSUE THREAD

**Issue by tunjid** [tunjid/heron#1316]
See https://github.com/tunjid/heron/issues/1311

The `MediaCreatorAndDescription` Composable for the [GalleryScreen](https://github.com/tunjid/heron/blob/93543b59396465ccb240220d7034562092354341/feature/gallery/src/commonMain/kotlin/com/tunjid/heron/gallery/ui/Media.kt#L211) should have another item for the media alt text if available.

We should display the alt text in an elevated card to make it distinct from the caption of the post.
